service-wfs-client/src/main/java/org/oskari/service/wfs/client/OskariWFSHttpUtil.java

```diff
@@ -0,0 +1,54 @@
+package org.oskari.service.wfs.client;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.Collections;
+import java.util.Map;
+
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import fi.nls.oskari.util.IOHelper;
+
+public class OskariWFSHttpUtil {
```

Comment:
This doesn't really have anything WFS-specific. Add these to IOHelper so other functionalities benefit from the redirect following as well?

Reply:
Sure